m/mysql-connector-j/mysql-connector-j_ubi_8.5.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package       : mysql-connector-j
+# Version       : 8.0.28
+# Source repo	: https://github.com/mysql/mysql-connector-j.git
+# Tested on     : UBI 8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vikas Kumar <kumar.vikas@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=mysql-connector-j
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-8.0.28}
+PACKAGE_URL=https://github.com/mysql/mysql-connector-j.git
+
+yum install -y git wget java-1.8.0-openjdk-devel
+
+cd /root
```

Comment:
do we need to run as root? does this need to be hard coded?

Reply:
The hard-coded path has been removed.